server/base/src/main/java/org/apache/accumulo/server/cli/ClientOpts.java

```diff
@@ -19,13 +19,18 @@
 import org.apache.accumulo.core.client.AccumuloException;
 import org.apache.accumulo.core.client.AccumuloSecurityException;
 import org.apache.accumulo.core.client.Instance;
+import org.apache.accumulo.core.client.impl.ClientContext;
 import org.apache.accumulo.server.client.HdfsZooInstance;
 
 public class ClientOpts extends org.apache.accumulo.core.cli.ClientOpts {
   {
     setPrincipal("root");
   }
 
+  public ClientContext getClientContext() {
```

Comment:
Not objecting to this, but what's this used for? Can it completely replace the other methods, like `getInstance()`?

Reply:
It's there to replace usages of `getInstance` but I haven't replaced them all yet.